contributing_to_docs/doc_guidelines.adoc

```diff
@@ -911,22 +911,134 @@ kind: Pod
 metadata:
   labels:
     test: liveness
-...
+# ...
 ----
 +
 or
 +
 ----
-...
+# ...
 spec:
   containers:
   - args:
     image: k8s.gcr.io/liveness
-...
+# ...
 ----
 +
 Do not use `[...]`, `<snip>`, or any other variant.
 
+* Do not use `[source,yaml]` for the following:
+
+** Bash "here" documents, such as the following example:
++
+....
+[source,terminal]
+----
+$ cat <<EOF| oc create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mlistener
+  labels:
+    app: multicast-verify
+EOF
+----
+....
+
+** The output of `oc describe <pural> <object>`, such as in the following example:
++
+....
+[source,text]
```

Comment:
This is actually news to me. Don't we do source,terminal for anything that is output from a CLI command? Just curious what is special with this one.

Reply:
@bergerhoffer, the value of `[source,text]` is that it literally does nothing. With `[source,terminal]` if the output starts with `#` for example, it will get erroneous highlighting. There could be other characters that might trigger it as well, not sure, but with `[source,text]` vanilla text output will always be unhighlighted. And I think in general, for the output of a command, we probably don't want to unintentionally highlight anything.